### ISSUE TITLE
Exiting the Game Triggers an Unwanted Replay

### ISSUE THREAD

**Issue by curif** [curif/AgeOfJoy-2022.1#551]
Geometrizer — 

I was testing the game with some other people last night and all of them ran into this issue 

When you hold both triggers to exit, if your hands are in close proximity to each other, you pick up a coin. After the timer is finished and the game quits, you still have the coin in your hand, and when you release the triggers, it can very easily drop the coin back into the arcade machine and start it again.
So basically people kept getting stuck in loops where they would exit the game and then automatically drop a coin into it again to restart it and they didn't know why because they weren't looking at their hands
I think the solution to this is, when you exit the game, you should automatically clear out the coin that is in your hand
